### ISSUE TITLE
Dynamic children height

### ISSUE THREAD

**Issue by kinhunt** [maxs15/react-native-modalbox#29]
Hi, cool project. what if the height of children is dynamic. how to display them in the center with backdrop?

**Comment by maxs15** [comment]
One solution would be to use the height/top/bottom properties to control the size and position of the modal.

**Comment by maxs15** [comment]
This is related to #28

**Comment by braco** [comment]
Solved this with the following hacky code
```
export default class Foo extends Component {
  static initialState = {
    style: {}
  };
  resizeModal(ev) {
    this.setState({style: {height: ev.nativeEvent.layout.height + 10}});
  }
  render() {
    return (
      <Modal style={[styles.modal, this.state.style]}>
        <View style={styles.inner} onLayout={(ev)=>{this.resizeModal(ev)}}>
          // Content here
        </View>
      </Modal>
   );
  }
}
```

**Comment by tonatiuh** [comment]
Worked like a charm, thanks @braco. 

I think this should be added somewhere in the documentation.

**Comment by shirbr510** [comment]
is there any progress besides @braco 's hacky solution?
(which btw, works like charm)

**Comment by sutinghui24** [comment]
just set height null

**Comment by alexzobi** [comment]
I tried the above workarounds without any luck. My modal is pinned to the bottom but it was cutting off content no matter what i did. I had position set to bottom because I wanted the modal tacked to the bottom of the screen, but I could only get chopped content. Then I realized something. I had not set coverscreen.  the solution for me was the following: 

coverscreen={true}, 
backgroundOpacity={0} (I didn't want to hide the background)
style={{height: null}}
